packages/manager/pages/index.tsx

```diff
@@ -1,10 +1,12 @@
 import Body from "../components/layout/body";
 import { ComponentWithAuth } from "../utils/auth.utils";
+// import '@one-platform/opc-feedback/dist/opc-feedback';
```

Comment:
can you please remove this, if we are not using this in future. If not then it's okay